docs/_jsdoc/index.md

```diff
@@ -1,8 +1,8 @@
-[![Build Status](https://dev.azure.com/Hyperledger/Fabric-Chaincode-Node/_apis/build/status/Fabric-Chaincode-Node?branchName=main)](https://dev.azure.com/Hyperledger/Fabric-Chaincode-Node/_build/latest?definitionId=33&branchName=main)
+[![Build Status](https://img.shields.io/github/actions/workflow/status/hyperledger/fabric-chaincode-node/pull_request.yaml?branch=main&label=build%20-%20main)](https://github.com/hyperledger/fabric-chaincode-node/actions/workflows/pull_request.yaml)
```

Comment:
This one shows for me as "No status". Maybe it should use this?

```suggestion
[![Build Status](https://github.com/hyperledger/fabric-chaincode-node/actions/workflows/schedule.yaml/badge.svg)](https://github.com/hyperledger/fabric-chaincode-node/actions/workflows/schedule.yaml)
```

Reply:
Done!